incubator/hnc/cmd/manager/main.go

```diff
@@ -86,6 +88,8 @@ func main() {
 	flag.IntVar(&qps, "apiserver-qps-throttle", 50, "The maximum QPS to the API server.")
 	flag.BoolVar(&stats.SuppressObjectTags, "suppress-object-tags", true, "If true, suppresses the kinds of object metrics to reduce metric cardinality.")
 	flag.IntVar(&webhookServerPort, "webhook-server-port", 443, "The port that the webhook server serves at.")
+	uaArg := arrayArg{val: &config.UnpropagatedAnnotations}
+	flag.Var(&uaArg, "unpropagated-annotation", "An annotation that should not be propagated by HNC. May be specified multiple times.")
```

Comment:
I think this description could be clarified a bit to indicate the annotations are stripped from propagated resources. It currently could also be read as (IMO at least), if this annotation exists, the resources will not be propagated.

It may also be worthwhile to indicate restarting HNC with different values will in fact affect already propagated resources.

Reply:
sg - I'll add proper documentation and a link.

Reply:
Not sure if it should just use plural in the description "Annotations that..", since the var is an array itself.

Reply:
Any _one_ use of the flag is singular - you can just use the flag multiple times, e.g.:

`--unpropagated-annotation=acme.com/foo --unpropagated-annotation=acme.com/bar`

So singular is correct IMO, since the arg to this flag is only a single annotation.

Reply:
sg

Reply:
Added new documentation in https://github.com/adrianludwin/multi-tenancy/blob/unprop-args/incubator/hnc/docs/user-guide/how-to.md#admin-cli-args and tweaked the descriptions as described.